webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -749,6 +749,10 @@ Kommunicate.richMsgEventHandler = {
         console.log('passenger detail submitted');
     },
     processQuickReplies: function (e) {
+        eventMapping.onRichMessageButtonClick.eventLabel = e.target.innerText;
```

Comment:
as onRichMessageButtonClick is global, so setting .eventLabel is problematic here.
what if at the same time some other event also happens then eventLabel value will be overriden